data/text/text_1.asm

```diff
@@ -71,29 +71,29 @@ _NoMoreRoomForItemText::
 
 _OaksAideHiText::
 	text "היי! זוכר אותי?"
-	line "אני העוזר של פרופ'"
-	cont "אלון!"
+	line "אני העוזר של"
+	cont "פרופ' אלון!"
 
 	para "אם תפסת @"
 	text_decimal hOaksAideRequirement, 1, 3
 	text ""
 	line "סוגים שונים של"
-	cont "#ימון, אלון ביקש"
-	cont "שאני אתן לך"
+	cont "#ימון, אלון"
+	cont "ביקש שאתן לך"
 	cont "@"
 	text_ram wOaksAideRewardItemName
 	text "!"
 
 	para "<PLAYER>! כבר"
 	line "תפסת לפחות @"
 	text_decimal hOaksAideRequirement, 1, 3
-	text "סוגי #ימון שונים?"
+	text "סוגי #ימון שונים?" ; TODO add another line
```

Comment:
```suggestion
	text "סוגי #ימון"
	cont "שונים?"
```

Reply:
כשניסיתי את זה ב-#137 זה לא עבד. אם נמצא איך לפתור את זה, יהיה אפשרי לעשות את אותו הדבר כאן

Reply:
אז אני אאשר, אמזג כדי שתוכל לפתוח PR נוסף שאאשר אותו (אם אני אפתח את PR אני לא אוכל לאשר אותו)

Reply:
Let me shed some light on the situation: after a `text` command, you may ONLY use the `line` command. The `text` command will never be followed by the `cont` command. Therefore, your solution would be:
```suggestion
	text "סוגי #ימון"
	line "שונים?"
```